transcribing/file.cs

```
@@ -1,144 +1,73 @@
 using System;
-using System.Collections.Generic;
 using System.IO;
 using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Net.Http.Json;
 using System.Text;
-using System.Threading;
+using System.Text.Json;
 using System.Threading.Tasks;
-using Newtonsoft.Json;
-using Newtonsoft.Json.Linq;
 
-public class FileUploader
+public class Program
 {
-    public async Task<string> UploadFileAsync(string apiToken, string path)
-    {
-        using var client = new HttpClient();
-        client.DefaultRequestHeaders.Authorization = new System.Net.Http.Headers.AuthenticationHeaderValue(apiToken);
-
-        using var fileContent = new ByteArrayContent(File.ReadAllBytes(path));
-        fileContent.Headers.ContentType = new System.Net.Http.Headers.MediaTypeHeaderValue("application/octet-stream");
+    private static readonly HttpClient HttpClient = new HttpClient();
 
-        HttpResponseMessage response;
-        try
-        {
-            response = await client.PostAsync("https://api.assemblyai.com/v2/upload", fileContent);
-        }
-        catch (Exception e)
-        {
-            Console.Error.WriteLine($"Error: {e.Message}");
-            return null;
-        }
```

No need for all the try/catches. If an exception is thrown, let it bubble up to the root and handle it there, or not. If an exception isn't caught, it should be written to the console automatically, so catching it yourself and then logging it to the console doesn't make much sense.